Chore [Docs]: Http Chaos Latency experiment docs (#3619)
litmuschaos/litmus: mkdocs/docs/experiments/categories/pods/pod-http-latency.md

          kind: Role
          name: pod-http-latency-sa
        subjects:
        - kind: ServiceAccount
          name: pod-http-latency-sa
          namespace: default
        ```
        Use this sample RBAC manifest to create a chaosServiceAccount in the desired (app) namespace. This example consists of the minimum necessary role permissions to execute the experiment.

## Experiment tunables

??? info "check the experiment tunables"
    <h2>Mandatory Fields</h2>

    <table>
      <tr>
        <th> Variables </th>
        <th> Description </th>
        <th> Notes </th>
      </tr>
      <tr>
        <td> TARGET_PORT </td>
        <td> Port of the service to target</td>
        <td>Defaults to port 80 </td>
      </tr>
    </table>

    <h2>Optional Fields</h2>

    <table>
      <tr>
        <th> Variables </th>
        <th> Description </th>
        <th> Notes </th>
      </tr>
      <tr>
        <td> LISTEN_PORT  </td>
        <td> Port where the proxy will be listening for requests</td>
        <td> Defaults to 20000 </td>
      </tr>
      <tr>
        <td> LATENCY  </td>
        <td> Latency value in ms to be added to requests</td>

>         <td> Defaults to 2000 </td>

Reviewer: We should add this as mandatory

      </tr>
      <tr>
        <td> CONTAINER_RUNTIME  </td>
        <td> container runtime interface for the cluster</td>
        <td> Defaults to docker, supported values: docker, containerd and crio for litmus and only docker for pumba LIB </td>
      </tr>
      <tr>
        <td> SOCKET_PATH </td>
        <td> Path of the containerd/crio/docker socket file </td>
        <td> Defaults to `/var/run/docker.sock` </td>
      </tr>
      <tr>
        <td> TOTAL_CHAOS_DURATION </td>

>         <td> The time duration for chaos insertion (seconds) </td>

Reviewer: ```suggestion
        <td> The duration of chaos injection (seconds) </td>
```

## Introduction

- It injects http request latency on the specified container by starting proxy server and then redirecting the traffic to the proxy server.
- It can test the application's resilience to lossy/flaky http requests

!!! tip "Scenario: Corrupt the network packets of target pod"    

>     ![Pod Network Corruption](../../images/network-chaos.png)

Reviewer: Since the Chaos Variables are different for this experiment, can you add a new image? For the sake of uniformity, use draw.io

Author: Have updated the image now

## Introduction

- It injects http response latency on the service whose port is provided as `TARGET_PORT` by starting proxy server and then redirecting the traffic through the proxy server.
- It can test the application's resilience to lossy/flaky http responses.

!!! tip "Scenario: Add latency to the HTTP request"    
    ![Pod HTTP Latency](../../images/pod-http.png)

## Uses

??? info "View the uses of the experiment" 
    coming soon

## Prerequisites

??? info "Verify the prerequisites" 

>     - Ensure that Kubernetes Version > 1.16 

Reviewer 2: ```suggestion
    - Ensure that Kubernetes Version > 1.17
```
